beta/aws-ecsfargate-cfn/README.md

```diff
@@ -0,0 +1,220 @@
+- 2 public subnets (each with `/20` CIDR blocks within this range), spanned across two availability zones in the current or selected region
+- an internet gateway
+- an Application Load Balancer (ALB)
+- an ECS task with associated container definitions for 1Password SCIM Bridge and the requred Redis cache
+- an ECS service and ECS cluster for the above task
+- a target group targeting the SCIM bridge container
+- 2 security groups to restrict traffic to and from the ALB and the ECS cluster, respectively
+- an AWS Certificate Manager (ACM) TLS certificate for the ALB
+- 2 Route 53 DNS `A` records:
+  - for the domain name of your SCIM bridge (e.g. `scim.example.com`)
+  - to validate the requested ACM certificate
+- an AWS Secrets Manager secret to store the `scimsession` credentials for your SCIM bridge
+- a CloudWatch log group to capture logs in streams from each of the SCIM bridge and Redis containers
+- required IAM roles
+
+This template is a working example to be used as a base for your SCIM bridge deployment and may need to be customized to meet your specific requirements.
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+  > **Note**
+  >
+  > 👀 Try 1Password Business **free** for 14 days: <https://start.1password.com/sign-up/business>
+- An AWS account with the permissions and available quota to create and manage the described resources
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) (if you want to manage the deployment using a terminal or script).
+
+## Getting started
+
+Before deploying 1Password SCIM Bridge, consult the [Preparation Guide](/PREPARATION.md) in this repository. Since this CloudFormation template will create all necessary resources, you do not need to seperately create a DNS record for your SCIM bridge, nor use its certificate manager component to create and manage a TLS certificate.
+
+1. Clone this repository and switch to this directory:
+
+   ```sh
+   git clone https://github.com/1Password/scim-examples.git
+   cd ./scim-examples/beta/aws-ecsfargate-cfn
+   ```
+
+   or, download the template file [`op-scim-bridge.yaml`](./op-scim-bridge.yaml) to a working directory on your computer.
+2. Follow the steps in [Automate provisioning in 1Password Business using SCIM](https://support.1password.com/scim/#step-1-set-up-and-deploy-1password-scim-bridge) to generate credentials for your SCIM bridge.
+3. Save the `scimsession` credentials file and the associated bearer token as items in your 1Password account.
+4. Download the `scimsession` file to the same working directory.
+
+## 🏗️ Deploy 1Password SCIM Bridge
+
+Use the AWS CloudFormation console to [create a stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stack.html) or deploy 1Password SCIM Bridge [using AWS CLI](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-cli.html):
```

Comment:
I expected these links to link to the sections below, rather than to AWS' docs. Would it be worth making the H3 subsections here collapsable so the reader can quickly choose their method?

Reply:
Good call. I'm trying to be thorough and link back to the reference documentation but that also makes sense to me.

```suggestion
Create a stack for your SCIM bridge [in the AWS CloudFormation console](#-use-the-aws-cloudformation-console) or [using AWS CLI](#-use-aws-cli):
```

I'm more iffy about them being collapsible. I don't like hiding all the instructions away by default. 🤔